.drone.star

```diff
@@ -387,7 +390,7 @@ def notification():
 
     for channel, params in notify_channels.items():
         settings = {
-            "webhook": from_secret("private_rocketchat"),
+            "webhook": from_secret("ROCKETCHAT_WEBHOOK"),
```

Comment:
Does anybody still use or depend on Rocket.Chat notifications? If not, we might just get rid of this altogether.

Reply:
> Does anybody still use or depend on Rocket.Chat notifications? If not, we might just get rid of this altogether.

Nightlies are usually reported to chat. Also, when tag/release is done to give the confirmation that everything went well.